riscv/ns16550.cc

```diff
@@ -170,6 +171,9 @@ bool ns16550_t::load(reg_t addr, size_t len, uint8_t* bytes)
   if (reg_io_width != len) {
     return false;
   }
+  if (addr >= PGSIZE) {
```

Comment:
This works because `reg_io_width` and `len` are always 1 here, but perhaps this should match the style in the other devices `addr + len > PGSIZE`

Reply:
Agreed.